### DISCUSSION TITLE
Swipe gesture on phone doesn't work after i open fancybox

### DISCUSSION THREAD

**Discussion by niko3573** [fancyapps/ui#399] [Q&A]
Hi.

I have implemented fancyapps carousel with fancybox inside on this page: https://www.jensenvisual.dk/funkishus

It works as it should on pc, and it also works when I'm swiping through the carousel on my phone, 
but when I click on the carousel and swipe through the photos and close it again, the swipe gesture won't work anymore..

The only thing that works are the navigation buttons in the carousel, and if I touch the screen it stops the slide-animation.

Any ideas how I can get the touch working after I've opened the fancybox and closed it again?

**Comment by fancyapps** [comment]
Hi,

Could you please upgrade to v5 and see if the problem persists?
